<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1BDQC6B" ignored="0" name="Add Customer Test" tests="3" failures="3" timestamp="07 Jun 2020 17:03:30 GMT" time="9.238" errors="0">
  <testcase name="addCustomerTest" time="3.418" classname="testcases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="Add Customer Test failed">
      <![CDATA[java.lang.AssertionError: Add Customer Test failed
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:26)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase name="addCustomerTest" time="2.27" classname="testcases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="Add Customer Test failed">
      <![CDATA[java.lang.AssertionError: Add Customer Test failed
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:26)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase name="addCustomerTest" time="2.416" classname="testcases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="Add Customer Test failed">
      <![CDATA[java.lang.AssertionError: Add Customer Test failed
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:26)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
</testsuite> <!-- Add Customer Test -->
